plugin/evm/service.go

```diff
@@ -43,3 +45,17 @@ func (api *SnowmanAPI) IssueBlock(ctx context.Context) error {
 	api.vm.builder.signalTxsReady()
 	return nil
 }
+
+func (api *SnowmanAPI) GetSignature(ctx context.Context, signatureRequest *message.SignatureRequest) (*message.SignatureResponse, error) {
```

Comment:
I used the methods in [plugin/evm/admin](https://github.com/ava-labs/subnet-evm/blob/master/plugin/evm/admin.go) as examples that accept and return JSON. I also [this](https://github.com/ava-labs/subnet-evm/blob/master/rpc/service.go#L125) comment, but was unable to locate the referenced documentation.

Reply:
Can we make this a separate service and add it under the extension `/warp` ?